src/main/io/gps.c

```diff
@@ -517,6 +510,7 @@ void gpsUpdate(timeUs_t currentTimeUs)
                 gpsData.baudrateIndex %= GPS_INIT_ENTRIES;
             }
             gpsData.lastMessage = millis();
+            gpsData.airborneMode = false;
```

Comment:
This will prevent setting mode if communication is lost when armed. `enum` with unset/disarmed/armed is one way to fix this

Reply:
If communication is lost, then once it's re-established a full initialization will be done, and after that, if the quad is armed the airborne mode will be set again. I don't see an issue here.

Reply:
yes, I overlooked that. But then it means that full initialization is done when switching to pedestrian mode, it is probably better to send only mode-relate message in GPS_RECEIVING_DATA

Reply:
No, I added a comment above the sizeof(ubloxAirborne) to explain that we are only sending the amount of bytes of a CFG-NAV5 message. I know, it's a bit hacky. :-)